src/core.js

```diff
@@ -142,8 +142,11 @@ function repeatStr(n, str) {
   return new Array(n + 1).join(str || ' ');
 }
 
-const hint = new HTMLHint();
+const HTMLHint = new HTMLHintCore();
```

Comment:
It's a bit wired to see a `const` with _UPPERCapitalcase_  
Especially when using the instance, it confuses me, because I would thing `HTMLHint` is a class and I need to use `new HTMLHint()`  
Otherwise `HTMLHint.addRule` may be a **static** method of `HTMLHint`

This breaks so much naming conventions 😅

Reply:
Yes I know, but I didn't want to have to change everything yet. Wanted to clean / fix first and then start improving the code.